prow/oss/plugins.yaml

```diff
@@ -20,6 +20,16 @@ config_updater:
       name: plugins
     prow/prowjobs/**/*.yaml:
       name: job-config
+    prow/knaitve/config.yaml: # For knative prow
+      name: config
+      clusters: 
+        knative-prow-trusted:
```

Comment:
Where do these cluster keys come from?

Reply:
It'll be from `kubeconfig` secret, it'll need to be configured by oss-test-infra prow admin

Reply:
The kubeconfig for scheduling jobs? This isn't scheduling jobs though.

Reply:
Yeah it's not for scheduling jobs, the `kubeconfig` is mentioned as documented in updateconfig plugin:
```
# specify the clusters and namespaces that the configmap targets
      # which requires that either --build-cluster or --kubeconfig arg is enabled for Hook
      # https://github.com/kubernetes/test-infra/blob/master/prow/getting_started_deploy.md#run-test-pods-in-different-clusters
      # if not set or empty, it uses the cluster where prow components are running
      # and the specified namespace(s)
      # if defined, the above namespace will be ignored
      clusters: 
        others:
        - namespace1
```

My understanding is it reuses the same kubeconfig for hook
https://github.com/kubernetes/test-infra/tree/master/prow/plugins/updateconfig